apps/web/app/(public)/(projects)/projects/projects-page.tsx

```
@@ -65,11 +73,110 @@ export default function ProjectsPage() {
+          <div className="text-center text-sm text-red-700">Error loading projects</div>
+        ) : data && data.data.length > 0 ? (
+          <div className="grid grid-cols-1 gap-8 sm:grid-cols-2">
+            {data.data.map((project) => (
+              <ProjectCard key={project.id} project={project} />
+            ))}
+          </div>
+        ) : (
+          <div className="text-center text-sm">No projects found</div>
+        )}
+
+        {data && data.pagination && data.pagination.totalPages > 1 && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)